modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java

```diff
@@ -1317,6 +1318,27 @@ public void testUpdateCustomObjectField() throws Exception {
 		Assert.assertEquals("baker", objectField.getName());
 		Assert.assertTrue(objectField.isRequired());
 
+		// Object field label needs to be replicated when there is an update
+		// with another default language
+
+		LocaleUtil.setDefault(
+			LocaleUtil.GERMANY.getLanguage(), LocaleUtil.GERMANY.getCountry(),
+			LocaleUtil.GERMANY.getVariant());
+
+		objectField = _objectFieldLocalService.updateCustomObjectField(
+			StringPool.BLANK, objectField.getObjectFieldId(), 0,
+			ObjectFieldConstants.BUSINESS_TYPE_INTEGER,
+			ObjectFieldConstants.DB_TYPE_INTEGER, false, true, StringPool.BLANK,
+			objectField.getLabelMap(), false, objectField.getName(), false,
+			false, objectField.getObjectFieldSettings());
```

Comment:
@pedro-oliveira446 please check if you can use the existing method `_updateCustomObjectField`:

```
objectField = _updateCustomObjectField(objectField);
```